clients/ethereumjs/wrapper.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+client_datadir="~/.ethereum"
+
+client_args=("$@")
+while [[ $# -gt 0 ]]; do
+    case $1 in
+    --datadir=*)
+        client_datadir="${1#*=}"
+        ;;
+    --datadir)
+        client_datadir="${2}"
+        ;;
+    esac
+    shift
+done
+
+source /wrapper/wrapper.lib.sh
+
+start_client() {
+    source $testnet_dir/nodevars_env.txt
+
+    ephemery_args=""
+    if [ -z "$(echo "${client_args[@]}" | grep "bootnodes")" ]; then
+        ephemery_args="$ephemery_args --bootnodes=$BOOTNODE_ENODE_LIST"
+    fi
+
+    echo "args: ${client_args[@]} $ephemery_args"
+    node /usr/app/packages/client/dist/esm/bin/cli.js  --dataDir=$client_datadir --gethGenesis=$testnet_dir/genesis.json --rpcEngine --rpcEngineAddr 0.0.0.0 --jwtSecret=/execution-auth.jwt "${client_args[@]}" $ephemery_args
```

Comment:
please do not specify any static arguments here.
the wrapper images are intended to be drop-in replacements that work the same way as the original images.
depending on how these images are used, some of these flags might already be specified via `$client_args`, so you should only add them, if not already preset there. for example, see the [nethermind wrapper](https://github.com/pk910/ephemery-client-wrapper/blob/main/clients/nethermind/wrapper.sh#L24-L35).
engine flags should not be set at all, as that's normal client configuration that needs to be supplied for the normal images too